### ISSUE TITLE
generalize the library for any npm command

### ISSUE THREAD

**Issue by colthreepv** [balderdashy/enpeem#7]
This module could be an interactive npm command launcher, it just needs 2 changes:
- export `doNpmCommand()` like `npm.command`
- change `function doNpmCommand(options, cb) {` to `function doNpmCommand(command, options, cb) {` so it can be used like this `var npmLs = npm.custom.bind(null, 'ls')`

I have made some experiments on my own, this functionality seems very easy, for now I am not compiling a PR because there are many PR still pending and collabs/owner does not seem really responsive on that..

just reply if there is any interest in this feature

**Comment by sailsbot** [comment]
Thanks for posting, @colthreepv.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:
- review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
- create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
